src/test/java/com/ioet/bpm/organizations/BpmOrganizationsApiApplicationTests.java

```diff
@@ -0,0 +1,140 @@
+package com.ioet.bpm.organizations;
+
+import com.ioet.bpm.organizations.boundaries.OrganizationController;
+import com.ioet.bpm.organizations.domain.Organization;
+import com.ioet.bpm.organizations.repositories.OrganizationRepository;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.internal.matchers.Or;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class BpmOrganizationsApiApplicationTests {
```

Comment:
This class should be renamed as OrganizationControllerTest, you're testing the class OrganizationController not the class BpmOrganizationsApiApplication.